public/stylesheets/main.css

```diff
@@ -37,3 +48,115 @@ h3 {
 }
 
 /* TODO - Add your own styles here */
+
+.title h2 {
+
+  font-size:3em;
+  text-align: center;
+  padding-top:0.5em;
+
+  font-weight:700;
+
+}
+
+.content h3{
+ 
+
+}
+
+.section-one {
+
+  height: 100vh;
+  width:100vw;
+  background-color:#49a7e9;
+}
+
+.section-two, .section-three, .section-four, .section-five, .section-six, .section-seven, .section-eight{
+  height: 50vh;
+  width:100vw;
+  text-align: center;
+
+}
+
+.section-two, .section-six {
+
+  background-color: #fab125;
+
+}
+
+.section-two .content, .section-five .content, .section-six .content, .section-eight .content{
+  width: 70%;
+  margin:auto;
+  padding-top:1em;
+
+}
+
+.section-three, .section-seven{
+
+  background-color: #63BD2F;
+
+}
+
+.section-four, .section-eight{
+
+  background-color: #c869bf;
+
+}
+
+.section-five{
+
+  background-color: #0fbfcf;
+
+}
+
+.section-six{
+  margin:0 auto;
+}
+
+.section-six .form-group{
+
```

Comment:
Pay attention to your coding style. Try and be consistent with your use of whitespace and indentation as this helps your fellow developers scan and understand your code at a glance.